Phonebook.py

```diff
@@ -0,0 +1,72 @@
+Phonebook = {"Abbas"    : 317,
+             "Haider"   : 318,
+             "Ali"      : 319,
+             "Muhammad" : 320}
+
+
+def read_contact():
+    name = input("Enter name: ")
+    if name in Phonebook:
+        print(f"{name}: {Phonebook[name]}")
+    else:
+        print("Contact does not exist")
+
+def edit_contact():
+    name = input("Enter the name for edit: ")
+    if name in Phonebook:
+        number = input("Enter the number for edit: ")
+        Phonebook[name] = number
+        print("Edited the Phonebook")
+    else:
+        print("Contact does not exist")
+def delete_contact():
```

Comment:
```suggestion
        print("Contact does not exist")

def delete_contact():
```
Always have a newline after function definitions.